app/src/main/java/de/westnordost/streetcomplete/data/osm/osmquest/OsmElementQuestType.kt

```diff
@@ -6,14 +6,13 @@ import de.westnordost.streetcomplete.data.quest.QuestType
 import de.westnordost.streetcomplete.data.quest.AllCountries
 import de.westnordost.streetcomplete.data.quest.Countries
 import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.ktx.asSingleArray
 
 /** Quest type where each quest refers to an OSM element */
 interface OsmElementQuestType<T> : QuestType<T> {
 
-    fun getTitleArgs(tags: Map<String, String>, featureName: Lazy<String?>): Array<String> {
-        val name = tags["name"] ?: tags["brand"]
-        return if (name != null) arrayOf(name) else arrayOf()
-    }
+    fun getTitleArgs(tags: Map<String, String>, featureName: Lazy<String?>): Array<String> =
+        (tags["name"] ?: tags["brand"]).asSingleArray()
```

Comment:
Should we include `tags["operator"]` as a third fallback here?

Reply:
That (and network) would seem easier to me than trying to guess and keep up to date with what is used where. I assume it will just return the first one?

Reply:
Yes, `tags["name"]` returns either a `String` or `null`, and the `?:` operator evaluates to the right side if the left is `null`.

Reply:
No, we shan't because the operator could be "Max Mustermann". Only for certain elements it makes sense to display that as a "name" (f.e. for ATMs it may make sense)

Reply:
I was suggesting as the THIRD option, i.e. to fall back on if there is no name or brand.

All of these things in NSI are operator based:
https://nsi.guide/index.html?t=operators

Some, such as a lot of charging stations, don't have a brand set:
https://nsi.guide/index.html?t=operators&k=amenity&v=charging_station

Reply:
Or does this mean it's a choice between "Max Mustermann" or ATM? I thought from the other code if it had a name-like field and a feature type you'd get "Does this `ATM` (`Max Mustermann`) exist still?"